client/src/pages/login/loginRoute.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import { Switch, Route } from 'react-router-dom' //adds Switch and Route libraries to this file
+import Login from './login/login.jsx' //links Login page
+import Admin from './adminDash/admin.jsx' //links Admin Dashboard
+import App from './applicantDash/applicant.jsx' //Links Applicants Dashboard
+import Judge from './judgingDash/judging.jsx' //Links Judges Dashboard
+import Vol from './volDash/volDash.jsx' //Links Volunteers Dashboard
+
+function LoginRoute() {
```

Comment:
Hey Shiv, can you change the way your routing works to have a dedicated router that has paths defined in json format that point to each page for us?

[Example Routing System](https://github.com/SSites/Ozone/tree/master/client/src/routes)

Reply:
Yes, I'll look into this!

Reply:
Just implemented a central routing system. All routes go to a new `\routes\appRoutes.jsx' JSON file, `<Switch>`s for ReactJS are implemented in `Main.jsx` by mapping over the JSON array and pulling respective fields from the objects.